docs/content.rst

```diff
@@ -614,6 +614,14 @@ the ``DEFAULT_METADATA``::
 To publish a post when the default status is ``draft``, update the post's
 metadata to include ``Status: published``.
 
+Hidden posts
+=================
```

Comment:
Perhaps shorten this header underline to match the length of the header text above it?